### PR TITLE
Bug/fir 24712 time functions

### PR DESCRIPTION
**Pull request by alexradzin**
fixed almost all problems of timestamp support. One test is partially disabled for now.

### CODE REVIEW

**Review by ptiurin — APPROVED** [review]
lgtm, let's file a ticket on that commented out test so we don't forget to fix it.

File: src/integrationTest/java/integration/tests/TimestampTest.java [hunk 1]
```diff
@@ -279,7 +278,7 @@ private void compareAllDateTimeResultSetValuesWithPostgres(ResultSet fireboltRes
 		try (Connection postgresConnection = embeddedPostgres.getPostgresDatabase().getConnection();
 				Statement pgStatement = postgresConnection.createStatement()) {
 			if (timezone != null) {
-				pgStatement.execute(String.format("set timezone = '%s'", timezone));
```
Comment (stepansergeevitch) [inline src/integrationTest/java/integration/tests/TimestampTest.java:282]: Please explain why did we have to remove the equals sign?

**Review by stepansergeevitch — APPROVED** [review]
Generally LGTM, one single comment before merging

File: src/integrationTest/java/integration/tests/TimestampTest.java (hunk 1 above)
Comment (alexradzin) [reply src/integrationTest/java/integration/tests/TimestampTest.java:282]: Good catch. Fixed.

**Pull request merged**